@xen-orchestra/fs/src/local.js

```diff
@@ -92,7 +96,10 @@ export default class LocalHandler extends RemoteHandlerAbstract {
   }
 
   async _readFile(file, options) {
-    return fs.readFile(this._getFilePath(file), options)
+    return retry(() => fs.readFile(this._getFilePath(file), options), {
+      tries: this.tries,
```

Comment:
It's very minor, but do you think it's easier for users to reason about the number of tries or retries in that case?

Reply:
For me, it's easier to imagine "how many tries we want" instead of "how much retries if it failed"